### ISSUE TITLE
class properties for objective-C interfaces are not correctly generated

### ISSUE THREAD

**Issue by brianquinlan** [dart-lang/ffigen#597]
For example, this header code:
```
@interface NSURLCache : NSObject
{
    @private
    NSURLCacheInternal *_internal;
}

@property (class, strong) NSURLCache *sharedURLCache;
...
```

Generates this class:
```dart
class NSURLCache extends _ObjCWrapper {
  NSURLCache._(ffi.Pointer<ObjCObject> id, NativeCupertinoHttp lib,
      {bool retain = false, bool release = false})
      : super._(id, lib, retain: retain, release: release);

  /// Returns a [NSURLCache] that points to the same underlying object as [other].
  static NSURLCache castFrom<T extends _ObjCWrapper>(T other) {
    return NSURLCache._(other._id, other._lib, retain: true, release: true);
  }

  /// Returns a [NSURLCache] that wraps the given raw object pointer.
  static NSURLCache castFromPointer(
      NativeCupertinoHttp lib, ffi.Pointer<ObjCObject> other,
      {bool retain = false, bool release = false}) {
    return NSURLCache._(other, lib, retain: retain, release: release);
  }

  /// Returns whether [obj] is an instance of [NSURLCache].
  static bool isInstance(_ObjCWrapper obj) {
    return obj._lib._objc_msgSend_0(
        obj._id, obj._lib._sel_isKindOfClass_1, obj._lib._class_NSURLCache1);
  }
}
```

**Comment by liamappelbe** [comment]
I can't repro this. When I generate the interface for `NSURLCache` I do get the `getSharedURLCache` and `setSharedURLCache` methods. So maybe this was fixed in a more recent version of ffigen, or maybe there's something different about our environments.

**Comment by brianquinlan** [comment]
Interesting, let me try again and I'll be sure to use the latest version of `ffigen`

**Comment by brianquinlan** [comment]
I upgraded to ffigen 9.0.1 which has fixed the issue
